odh-jupyter-trash-cleanup/odh_jupyter_trash_cleanup/test_empty_trash.py

```
@@ -0,0 +1,52 @@
+
+import tempfile
+from pathlib import Path
+import os
+from odh_jupyter_trash_cleanup.trash import Trash
+
+def test_empty_directory_returns_zero():
+    with tempfile.TemporaryDirectory() as tmpdir:
+        temp_dir = Path(tmpdir) / "empty_dir"
+        os.mkdir(temp_dir)
+        result = Trash()._clear_dir(Path(temp_dir))
+        assert result == 0
```

Ruff (0.12.2)

Use of `assert` detected

(S101)